editor/scene_tree_editor.cpp

```diff
@@ -644,11 +644,12 @@ void SceneTreeEditor::_renamed() {
 	ERR_FAIL_COND(!n);
 
 	String new_name = which->get_text(0);
-	if (new_name.find(".") != -1 || new_name.find("/") != -1) {
+	if (Node::_validate_node_name(new_name) == false) {
```

Comment:
Just a style nitpick, but we tend to use `!cond` instead of `cond == false`.

Reply:
@akien-mga fixed.